samples/apps/copilot-chat-app/documentuploadapp/README.md

```diff
@@ -0,0 +1,49 @@
+﻿# Copilot Chat Document Upload App
+
+> **!IMPORTANT**
+> This learning sample is for educational purposes only and should not be used in any
+> production use case. It is intended to highlight concepts of Semantic Kernel and not
+> any architectural / security design practices to be used.
+
+### Watch the Copilot Chat Sample Quick Start Video [here](https://aka.ms/SK-Copilotchat-video).
+
+## Introduction
+
+One of the exciting features of the Copilot Chat App is its ability to store contextual information
+to [memories](https://github.com/microsoft/semantic-kernel/blob/main/docs/EMBEDDINGS.md) and retrieve
+relevant information from memories to provide more meaningful answers to users through out the conversations.
+
+Memories can be generated from conversations as well as injected from external sources, such as text documents.
+Ingesting documents to the service will make the App even more intelligent by having prior knowledge of very
+specific contexts that are generally unavailable publicly, such as enterprise data.
+
+## Requirements to run this app
+1. A registered App in Azure Portal (https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
```

Comment:
Does this also support no authentication? That's the default for the service right now.

Reply:
Yes, if you just want to upload the document to the global collection, then you don't need authentication.